docker-compose.yml

```diff
@@ -39,6 +39,8 @@ services:
             - '3000:3000'
         networks:
             - backend_network
+        env_file:
+            - .env
```

Comment:
What do you think about creating the .env file inside the app/backend dir? Then we could do it the same for the frontend .env. Additionally I would create a .postman.env, which contains just the information needed for the postgres docker image. => You would need to adjust configService as well, if you chose to do so. 
```ts

	imports: [
		ConfigModule.forRoot({
			isGlobal: true,
			envFilePath: ['.env'], // here
			validate: validateConfig,
		}),
```

Reply:
we could do that yes, i thought we would go for  one single .env, but i can refactor it

Reply:
Ok moved it to the backend Thanks for the tipp